### PR TITLE
Support marking dates before a min and max date as passive.

### PR DESCRIPTION
**Pull request by mattfurness**
Implements #33

### CODE REVIEW

File: src/Calendar.js [hunk 1]
```diff
@@ -180,6 +188,7 @@ class Calendar extends Component {
           isSelected={ isSelected || isEdge }
           isInRange={ isInRange }
           key={ index }
+          isPassive = { isPassive || isOutsideMinMax }
```
Comment (burakcan) [inline src/Calendar.js]: Is the `isPassive` a leftover? I couldn't find where its assigned.

**Comment by burakcan** [comment]
It looks like this is only to style passive days. Should we also prevent them to be selected? What do you think @mattfurness ?

**Comment by burakcan** [comment]
merged this too

File: src/Calendar.js (hunk 1 above)
Comment (mattfurness) [inline src/Calendar.js]: `isPassive` is a property on the `data` parameter that is passed into the `map` function. 

`const { dayMoment, isPassive } = data`.

**Comment by mattfurness** [comment]
Hi @burakcan,

Sorry for the late replies. If a `DayCell` has the `isPassive` prop the `handleSelect` function will return null.

`if (this.props.isPassive) return null;`

Cheers,
-Matt.

**Pull request merged**